frontend/packages/metal3-plugin/src/plugin.tsx

```diff
@@ -88,7 +88,7 @@ const plugin: Plugin<ConsumedExtensions> = [
     type: 'Page/Route',
     properties: {
       exact: true,
-      path: `/k8s/ns/:ns/${referenceForModel(BareMetalHostModel)}/~new/form`,
+      path: `/k8s/ns/:ns/${referenceForModel(BareMetalHostModel)}/~new-dialog`,
```

Comment:
can't we change the links to just `.../baremetalhosts/...` ? 

It is pretty mouthful: `/k8s/ns/openshift-machine-api/metal3.io~v1alpha1~BareMetalHost/openshift-master-0`

Reply:
`BareMetalHostModel` is CRD based, IIUC paths for CRD resources should use paths generated using `referenceForModel` to clearly identify the resource type.

Reply:
IMO we should strive for readability and not have overcomplicated URLs. I think we would be aware of such collision sooner than our users.